style.css

```diff
@@ -272,23 +277,28 @@ svg {
 .container-flex {
   display: flex;
   gap: 1rem;
+  justify-content: space-between;
+  align-items: center;
 }
 
 body {
   font-size: var(--fs-body);
   font-family: var(--ff-family-primary);
+  counter-reset: list;
 }
 
 /* unikamy stylizacji po nazwach ktore sa znacznikami */
-.button {
+.btn {
   border: none;
   border-radius: 6px;
   padding: 1.125em 2em;
   cursor: pointer;
   font-weight: var(--fw-super-bold);
   font-size: var(--fs-button);
 }
-
+.btn-shadow{
+  box-shadow: 0px 3px 9px rgba(57, 20, 0, 0.08);
```

Comment:
Nie wiem jak w dalszym designie, ale jeśli boxshadowy się powtarzają to warto umieścić je w zmiennej :)